neo4j/datadog_checks/neo4j/data/neo4j.yaml.example

```diff
@@ -0,0 +1,7 @@
+cluster_check: true
```

Comment:
What is this `neo4j.yaml.example` file for? If these are config options for `conf.yaml.example`, you can leave these as `example` values in [`spec.yaml`](https://github.com/DataDog/integrations-extras/blob/ccbb92ff6c29eadd8e63edbddd2409e4f092f697/neo4j/assets/configuration/spec.yaml) instead.